contracts/assemblyscript/flipper/package.json

```diff
@@ -12,11 +12,11 @@
     "build": "yarn build:optimized && yarn build:tsd"
   },
   "dependencies": {
-    "assemblyscript": "^0.8.1",
     "bignum": "MaxGraey/bignum.wasm"
   },
   "devDependencies": {
     "@types/node": "^10.17.2",
+    "assemblyscript": "^0.8.1-nightly.20191124",
```

Comment:
This essentially changes assemblyscript from stable to nightly. Is there a particular reason for that? If not, I'd prefer to stick to stable if possible.

Although I am not strong on this, since don't have a good understanding how they are differ though.

Reply:
nightly branch fixes "parenthesis issue" when method called immediately after constructor. Also was added ARC optimizations. It's pretty enough stable currently. Also I thought this repo has deeply WIP status which allow some experimental dependencies

Reply:
Ah, fair enough.